### PR TITLE
chore: light prover cleanup

### PR DESCRIPTION
**Pull request by ananas-block**
should be merged after https://github.com/Lightprotocol/light-protocol/pull/841

Changes:
- replace all ocurrences of utxo with compressed account in light prover

### CODE REVIEW

File: circuit-lib/circuit-lib.js/package.json [hunk 1]
```diff
@@ -47,6 +47,9 @@
   "nx": {
     "targets": {
       "build": {
+        "inputs": [
+          "{projectRoot}/cli"
```
Comment (sergeytimoshin) [inline circuit-lib/circuit-lib.js/package.json:51]:
```suggestion
          "{workspaceRoot}/cli"
```

File: cli/package.json [hunk 2]
```diff
@@ -121,5 +121,23 @@
   "keywords": [
     "oclif"
   ],
-  "types": "dist/index.d.ts"
+  "types": "dist/index.d.ts",
+  "nx": {
+    "targets": {
+      "build": {
+        "inputs": [
+          "{projectRoot}/js",
```
Comment (sergeytimoshin) [inline cli/package.json:129]:
```suggestion
          "{workspaceRoot}/js",
```

File: cli/package.json [hunk 3]
```diff
@@ -121,5 +121,23 @@
...
+          "{projectRoot}/programs",
```
Comment (sergeytimoshin) [inline cli/package.json:130]:
```suggestion
          "{workspaceRoot}/programs",
```

File: cli/package.json [hunk 4]
```diff
@@ -121,5 +121,23 @@
...
+          "{projectRoot}/gnark-prover",
```
Comment (sergeytimoshin) [inline cli/package.json:131]:
```suggestion
          "{workspaceRoot}/gnark-prover",
```

File: cli/package.json [hunk 5]
```diff
@@ -121,5 +121,23 @@
...
+          "{projectRoot}/forester"
```
Comment (sergeytimoshin) [inline cli/package.json:132]:
```suggestion
          "{workspaceRoot}/forester"
```

File: cli/src/utils/process.ts [hunk 6]
```diff
@@ -135,7 +135,7 @@ export function spawnBinary(command: string, args: string[] = []) {
 
   const out = fs.openSync(`${logDir}/${binaryName}.log`, "a");
   const err = fs.openSync(`${logDir}/${binaryName}.log`, "a");
-
+  console.log("Spawning binary: ", command, args);
```
Comment (sergeytimoshin) [inline cli/src/utils/process.ts:138]: Leftover?

File: .github/workflows/light-sdk-tests.yml [hunk 7]
```diff
@@ -38,6 +38,11 @@ jobs:
       - name: Setup and build
         uses: ./.github/actions/setup-and-build
 
+      - name: Build CLI
+        run: |
+          source ./scripts/devenv.sh
+          npx nx build @lightprotocol/zk-compression-cli --skip-nx-cache
```
Comment (sergeytimoshin) [inline .github/workflows/light-sdk-tests.yml:44]: Why do we need to build cli explicitly?

**Review by sergeytimoshin — DISMISSED** [review]

File: .github/workflows/light-sdk-tests.yml (hunk 7 above)
Comment (ananas-block) [reply .github/workflows/light-sdk-tests.yml:44]: probably a remainder of my cache debugging attempts, will remove it.

Comment (ananas-block) [reply .github/workflows/light-sdk-tests.yml:44]:
```suggestion
          npx nx build @lightprotocol/zk-compression-cli
```

File: cli/src/utils/process.ts (hunk 6 above)
Comment (ananas-block) [reply cli/src/utils/process.ts:138]:
```suggestion
```

**Review by SwenSchaeferjohann — APPROVED** [review]

**Pull request merged**